### ISSUE TITLE
Trying to use <color_fg_bg> tags causes a CTD

### ISSUE THREAD

**Issue by Procyonae** [CleverRaven/Cataclysm-DDA#77747]
### Describe the bug

get_color_from_tag() eventually calls color_from_string() which explicitly supports <color_fg_bg> but trying to use such tags like the examples below results in an error which when dismissed causes a CTD

### Attach save file

N/A

### Steps to reproduce

1. Add a <color_fg_bg> tag like <color_light_cyan_light_blue> to an existing "description" or "ascii_art" (I were using "battery_car")
2. Try to examine "battery_car" in game
3. Error prompt
4. Pick your poison
5. CTD

### Expected behavior

Pretty stuff :c

### Screenshots

![image](https://github.com/user-attachments/assets/474e6556-c5db-4940-b127-38514047a599)
Abort or Retry results in CTD, Ignore results in a new identical error prompt

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.5011 (22H2)
- Game Version: cdda-experimental-2024-11-09-1800 a05e593bf5 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

### Additional context

Boondiggle on the devcord was having problems with REXPaint's CDDA export function so I figured I'd write a python script that converts its save format straight to a complete json object and runs the formatter on it so we don't have to rely on the closed source being kept up to date with any changes to structure we require so I can provide a good test ascii art with every fg/bg combo for any PR solving this or unit test we add to stop this breaking again.

**Comment by mqrause** [comment]
I didn't track down why exactly the crash happens, but it looks like `color_light_cyan_light_blue` is missing in `color_manager::load_default`. And other combinations might be missing as well.

**Comment by mqrause** [comment]
I think the crash is related to debug messages showing in the middle of imgui drawing, so not directly related to the specific error. See #77763 for a different but similar case.

**Comment by db48x** [comment]
Yes, anything that causes an error while in the middle of an ImGui window is just going to cause a second error from ImGui itself, which is super annoying. I think there was already a bug filed specifically about that, but I don’t recall the title or the number offhand.
